### ISSUE TITLE
Branding documentation missing

### ISSUE THREAD

**Issue by tboerger** [owncloud/docs#408]
## What Type Of Content Change Is This?

- [x] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To?

- [ ] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [x] Branding Manual

## What Needs to be Documented?
We have not migrated the client branding documentation yet, this got to be migrated for the final switch to Antora.

## Why Should This Change Be Made?
It's an important missing part for our Antora migration.

**Comment by settermjd** [comment]
Is this what you're referring to? 

![bildschirmfoto 2018-11-22 um 16 41 04](https://user-images.githubusercontent.com/196801/48912311-8c771980-ee75-11e8-84c5-b0d92f6fc915.png)

As with the appliance issue, should this be moved so it's easier to find?

**Comment by tboerger** [comment]
Right, for server branding, but I'm talking about https://github.com/owncloud/branded_clients

**Comment by mmattel** [comment]
Could we then change the title of this issue from:
``Branding documentation missing`` --> ``Client Branding documentation missing``

**Comment by settermjd** [comment]
The branded_clients repo's [been migrated to Antora](https://github.com/owncloud/branded_clients/tree/master-antora)?
